crypto/src/key/rschnorr/mod.rs

```diff
@@ -129,6 +134,27 @@ impl MLRistrettoPrivateKey {
     }
 }
 
+impl ChildNumber {
```

Comment:
This is not really idiomatic rust. We shouldn't have struct impl blocks outside their modules.

Reply:
Would just having a simple function `child_num_to_schnorrkel_chaincode` outside of an `impl` enough?

Reply:
There probably should be a trait for this, which you implement for the schnorrkel key.